packages/xo-server/src/api/host.js

```diff
@@ -1,17 +1,22 @@
 import { format, JsonRpcError } from 'json-rpc-peer'
+import { satisfies as versionSatisfies } from 'semver'
 
 // ===================================================================
 
 export async function set({
   host,
-
```

Comment:
Keep the empty line, it's unrelated to your changes and is here to separate the host to the props we are going to change.